<?xml version='1.0'?>
<ns2:test-suite xmlns:ns2='urn:model.allure.qatools.yandex.ru' start='1748357104318' stop='1748357194715'>
    <name>[SMOKE] Get servers endpoint</name>
    <title>[SMOKE] Get servers endpoint</title>
    <test-cases>
        <test-case start='1748357104318' status='failed' stop='1748357119384'>
            <name>put method should respond with status code 405 and an error message</name>
            <title>put method should respond with status code 405 and an error message</title>
            <labels/>
            <parameters/>
            <steps/>
            <attachments/>
            <failure>
                <message>Error: Endpoint is not responding. 5 attempts has been made with 3 seconds apart. Endpoint: https://127.0.0.1/v1/servers. Method: put</message>
                <stack-trace>    at next (/app/node_modules/@nordvpn/tests-backend-core/src/request.js:38:11)
                    at /app/node_modules/@nordvpn/tests-backend-core/src/request.js:34:16
                    at async UserContext.&lt;anonymous> (/app/tests/smoke/servers.spec.js:17:28)</stack-trace>
            </failure>
        </test-case>
        <test-case start='1748357119387' status='failed' stop='1748357134435'>
            <name>post method should respond with status code 405 and an error message</name>
            <title>post method should respond with status code 405 and an error message</title>
            <labels/>
            <parameters/>
            <steps/>
            <attachments/>
            <failure>
                <message>Error: Endpoint is not responding. 5 attempts has been made with 3 seconds apart. Endpoint: https://127.0.0.1/v1/servers. Method: post</message>
                <stack-trace>    at next (/app/node_modules/@nordvpn/tests-backend-core/src/request.js:38:11)
                    at /app/node_modules/@nordvpn/tests-backend-core/src/request.js:34:16
                    at async UserContext.&lt;anonymous> (/app/tests/smoke/servers.spec.js:17:28)</stack-trace>
            </failure>
        </test-case>
        <test-case start='1748357134437' status='failed' stop='1748357149473'>
            <name>patch method should respond with status code 405 and an error message</name>
            <title>patch method should respond with status code 405 and an error message</title>
            <labels/>
            <parameters/>
            <steps/>
            <attachments/>
            <failure>
                <message>Error: Endpoint is not responding. 5 attempts has been made with 3 seconds apart. Endpoint: https://127.0.0.1/v1/servers. Method: patch</message>
                <stack-trace>    at next (/app/node_modules/@nordvpn/tests-backend-core/src/request.js:38:11)
                    at /app/node_modules/@nordvpn/tests-backend-core/src/request.js:34:16
                    at async UserContext.&lt;anonymous> (/app/tests/smoke/servers.spec.js:17:28)</stack-trace>
            </failure>
        </test-case>
        <test-case start='1748357149474' status='failed' stop='1748357164553'>
            <name>del method should respond with status code 405 and an error message</name>
            <title>del method should respond with status code 405 and an error message</title>
            <labels/>
            <parameters/>
            <steps/>
            <attachments/>
            <failure>
                <message>Error: Endpoint is not responding. 5 attempts has been made with 3 seconds apart. Endpoint: https://127.0.0.1/v1/servers. Method: delete</message>
                <stack-trace>    at next (/app/node_modules/@nordvpn/tests-backend-core/src/request.js:38:11)
                    at /app/node_modules/@nordvpn/tests-backend-core/src/request.js:34:16
                    at async UserContext.&lt;anonymous> (/app/tests/smoke/servers.spec.js:17:28)</stack-trace>
            </failure>
        </test-case>
        <test-case start='1748357164555' status='failed' stop='1748357179636'>
            <name>should return a valid response with an invalid type filter parameter</name>
            <title>should return a valid response with an invalid type filter parameter</title>
            <labels/>
            <parameters/>
            <steps/>
            <attachments/>
            <failure>
                <message>Error: Endpoint is not responding. 5 attempts has been made with 3 seconds apart. Endpoint: https://127.0.0.1/v1/servers. Method: get</message>
                <stack-trace>    at next (/app/node_modules/@nordvpn/tests-backend-core/src/request.js:38:11)
                    at /app/node_modules/@nordvpn/tests-backend-core/src/request.js:34:16
                    at async UserContext.&lt;anonymous> (/app/tests/smoke/servers.spec.js:35:43)</stack-trace>
            </failure>
        </test-case>
        <test-case start='1748357179639' status='failed' stop='1748357194707'>
            <name>should only return online servers</name>
            <title>should only return online servers</title>
            <labels/>
            <parameters/>
            <steps/>
            <attachments/>
            <failure>
                <message>Error: Endpoint is not responding. 5 attempts has been made with 3 seconds apart. Endpoint: https://127.0.0.1/v1/servers. Method: get</message>
                <stack-trace>    at next (/app/node_modules/@nordvpn/tests-backend-core/src/request.js:38:11)
                    at /app/node_modules/@nordvpn/tests-backend-core/src/request.js:34:16
                    at async UserContext.&lt;anonymous> (/app/tests/smoke/servers.spec.js:47:26)</stack-trace>
            </failure>
        </test-case>
    </test-cases>
</ns2:test-suite>